varcode/variant.py

```diff
@@ -681,3 +681,24 @@ def _exonic_transcript_effect(self, exon, exon_number, transcript):
                 exon=exon,
                 alternate_effect=coding_effect_annotation)
         return coding_effect_annotation
+
+    @property
+    def is_insertion(self):
+        """
+        Does this variant represent the insertion of nucleotides into the
+        reference genome?
+        """
+        return (len(self.ref) < len(self.alt)) and self.alt.startswith(self.ref)
```

Comment:
`self.alt.startswith(self.ref)` seems like a confusing thing to do when dealing with `""`? Am I misunderstanding?